docs/source/markdown/podman-machine-os-apply.1.md

```diff
@@ -0,0 +1,41 @@
+% podman-machine-os-apply 1
+
+## NAME
+podman\-machine\-os\-apply - Apply an OCI image to a Podman Machine's OS
+
+## SYNOPSIS
+**podman machine os apply** [*options*] *image* [vm]
+
+## DESCRIPTION
+
+Apply machine OS changes from an OCI image.
+
+VM's that use OS's that use rpm-ostreee have the capability to rebase itself from the content of an OCI image.
+`podman machine image apply` takes an OCI image with container native ostree functionality and rebases itself on that image.
+
+By default, Podman machines on Mac and Linux use an rpm-ostree based distrubition (Fedora CoreOS).
+
+For more information, please see the [rpm-ostree docs](https://coreos.github.io/rpm-ostree/container/).
```

Comment:
Is there an rpm-ostree manpage on this we can point to?

Reply:
This is the rpm-ostree docs, the `man` utility is not installed inside the podman VM, nor would the manpage exist on host systems, since this rpm-ostree is a command inside the VM. 

I think rpm-ostree does have manpages: https://github.com/coreos/rpm-ostree/blob/main/man/rpm-ostree.xml, but ATM I don't know how to access them.